keras/backend/tensorflow_backend.py

```diff
@@ -3154,6 +3154,27 @@ def in_top_k(predictions, targets, k):
 # CONVOLUTIONS
 
 
+def _preprocess_conv1d_input(x, data_format):
```

Comment:
Do we need a separate function? In `conv1d` we don't use one.

Reply:
@fchollet, Since `separable_conv1d` also need to check for `_has_nchw_support`, wouldn't it be nice if this was added?